ci/gpu/build.sh

```diff
@@ -62,6 +62,9 @@ gpuci_conda_retry install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvid
 # gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
 # gpuci_conda_retry install -y "your-pkg=1.0.0"
 
+gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
+gpuci_conda_retry install -c conda-forge "conda-forge::faiss=1.7.0" "conda-forge::faiss-proc=*=cuda"
```

Comment:
```suggestion
gpuci_conda_retry install -c conda-forge "conda-forge::libfaiss=1.7.0" "conda-forge::faiss-proc=*=cuda"
```

Fault of my advice, but by only stating faiss as oposed to libfaiss in the command, it gave freedom to conda to pull libfaiss from the pytorch channel with only cpu support, which happened in the CUDA 11 CI job:

```bash
    libfaiss:           1.7.0-h2bc3f7f_0_cpu                         pytorch  
```